### PR TITLE
Added support for i3en host instance type

### PR DESCRIPTION
**Pull request by saloneerege**

### CODE REVIEW

File: website/docs/r/cluster.html.markdown [hunk 1]
```diff
@@ -40,7 +40,7 @@ The following arguments are supported for vmc_cluster resource:
 
 * `host_cpu_cores_count` - (Optional) Customize CPU cores on hosts in a cluster. Specify number of cores to be enabled on hosts in a cluster.
 
-* `host_instance_type` - (Optional) The instance type for the esx hosts added to this cluster. Possible values are: I3_METAL, I3EN.metal and R5_METAL. Default value: I3_METAL.
+* `host_instance_type` - (Optional) The instance type for the esx hosts added to this cluster. Possible values are: I3_METAL, I3EN_METAL and R5_METAL. Default value: I3_METAL.
 
 * `storage_capacity` - (Optional) For EBS-backed instances i.e: for R5.METAL only, the requested storage capacity in GiB.
```
Comment (shylasrinivas) [inline website/docs/r/cluster.html.markdown]: should this be R5_METAL instead?

File: website/docs/r/sddc.html.markdown [hunk 2]
```diff
@@ -73,7 +73,7 @@ The following arguments are supported:
 
 * `account_link_sddc_config` - (Optional) The account linking configuration object.
 
-* `host_instance_type` -  (Optional) The instance type for the esx hosts in the primary cluster of the SDDC. Possible values : I3_METAL, R5_METAL.
+* `host_instance_type` -  (Optional) The instance type for the esx hosts in the primary cluster of the SDDC. Possible values : I3_METAL,I3EN_METAL and R5_METAL. Default value : I3_METAL. Currently I3EN_METAL host_instance_type does not support 1NODE and 2 node SDDC deployment.
```
Comment (shylasrinivas) [inline website/docs/r/sddc.html.markdown]: space after ,

File: website/docs/r/cluster.html.markdown (hunk 1 above)
Comment (saloneerege) [reply website/docs/r/cluster.html.markdown]: Yes. Good catch. I will modify it.

File: website/docs/r/sddc.html.markdown (hunk 2 above)
Comment (saloneerege) [reply website/docs/r/sddc.html.markdown]: Done.

File: website/docs/r/cluster.html.markdown (hunk 1 above)
Comment (saloneerege) [reply website/docs/r/cluster.html.markdown]: Done

**Review by shylasrinivas — APPROVED** [review]

**Review by sumitAgrawal007 — COMMENTED** [review]
LGTM

**Pull request merged**